File: experiments/llm-reasoning-cost/generated_code/java/03_vector_ops_mini.java
```java
import java.util.ArrayList;
import java.util.List;

public class VectorOperations {
    public static int sum_positive(List<Integer> numbers) {
        List<Integer> positiveNumbers = new ArrayList<>();
        for (int number : numbers) {
            if (number > 0) {
                positiveNumbers.add(number);
            }
        }
        int sum = 0;
        for (int positiveNumber : positiveNumbers) {
            sum += positiveNumber;
        }
        return sum;
    }

    public static void main(String[] args) {
        List<Integer> input = List.of(1, -2, 3, -4, 5);
        int result = sum_positive(input);
        System.out.println(result); // Output: 9
    }
}
```

File: experiments/llm-reasoning-cost/generated_code/go/03_vector_ops_mini.go
```go
package main

import "fmt"

func sum_positive(nums []int32) int32 {
    var positiveNums []int32
    for _, num := range nums {
        if num > 0 {
            positiveNums = append(positiveNums, num)
        }
    }
    
    var sum int32
    for _, num := range positiveNums {
        sum += num
    }
    
    return sum
}

func main() {
    input := []int32{1, -2, 3, -4, 5}
    result := sum_positive(input)
    fmt.Println(result) // Output: 9
}
```